docs/DEVELOPMENT.md

```diff
@@ -198,3 +198,39 @@ need to rerun Ansible to verify the configuration.
 
 To restore a backup, first download it. You can do this using the `aws s3` CLI.
 Then run `sudo /backups/scripts/restore-backup.sh <backup-file>`.
+
+## Code Linting using ReSharper CLI Tools and EditorConfig
```

Comment:
See this section for installing and using ReSharper.

Also see the links below to find all the options we can use in the `.editorconfig` file.  There are a lot of really granular settings we can tweak to our teams preferences.